Source/Render/client/D3DRenderDrawClient.cpp

```diff
@@ -49,7 +49,7 @@ void FFDData::SetIB(sPolygon* pIndex)
 	int xs = 1 + xsize;
 	int ys = ysize;
 
-	WORD *ib=(WORD*)pIndex;
+    int16_t *ib=(int16_t*)pIndex;

```

Comment:
Please also change WORD -> uin16_t in the below code: `int num=ib-(WORD*)pIndex;`